kodex-team-leadera.md

```diff
@@ -51,6 +51,10 @@ Neboj se to říct, nikdo není dokonalý. Řekni teamu, že si nejseš jistý,
 
 Stane se ti mnohokrát, že bude potřeba něco probrat se zákazníkem. Pokus se odhadnout situaci a vhodně jednat, aby si neudělal ostudu sobě ani Appliftingu. Ale zase se nepřetvařuj, máme svojí identitu. Když myslíme diplomata, nemyslíme populistu, spíš člověka slušného, s citem pro situaci.
 
+### Výjimečné situace
+
+Někdy je potřeba zabrat, protože se blíží termín dodání nebo se stal nějaký nepředvídaný problém. Na tobě je dohodnout se a ujasnit si s klientem jaká situace je považována za výjimečnou, se kterou mu moc rádi pomůžeme, ale bude to účtováno vyšší sazbou.
```

Comment:
Jelikož tohle budou číst i klienti, je potřeba víc vyspecifikovat, co je to výjmečná situace z naší strany. Overtimes? Dodatečná kapacita? Práce o víkendu?

Reply:
@Vratislav added, pls review